### PR TITLE
graphical improvements

### PR DESCRIPTION
**Pull request by sz1521**
- fix not updating first screen
- optimized first screen with "Loading" text
- use CRT effect as global effect (let's check performance!)
- add opacity effect to go
- add move keys info to start screen also (maybe needed or add one more press enter in next screen?)
- less brightness for pseudo 3d shadow

### CODE REVIEW

File: src/game.ts [hunk 1]
```diff
@@ -439,11 +412,34 @@ const drawStartScreen = (t: number, wait: boolean, z: number): void => {
...
+        data[i] = data[i + 1] = data[i + 2] = grayscale * 0.7;
+    }
+
+    // Put the modified image data back onto the canvas
+    cx.putImageData(imageData, 0, 0);
+}
+
 const drawInitialScreen = (): void => {
     cx.save();
-    cx.filter = "grayscale(1) brightness(0.6)";
```
Comment (sz1521) [inline src/game.ts:446]: This was slow on start up screen

File: src/Level.ts [hunk 2]
```diff
@@ -365,7 +365,7 @@ export class Level implements Area {
             cx.fillStyle = element.color;
             cx.shadowColor = element.color
                 .replace("rgb(", "rgba(")
-                .replace(")", ",0.5)");
+                .replace(")", ",0.3)");
```
Comment (sz1521) [inline src/Level.ts:368]: Maybe more clear what is actual trackelement and what not

File: src/game.ts [hunk 3]
```diff
@@ -136,40 +144,59 @@ const update = (t: number, dt: number): void => {
...
+            applyGrayscale();
+            applyCRTEffect(false);
+
+            break;
+        }
+        case GameState.Init: {
+            drawInitialScreen(true);
+            renderText(
+                "Press any key" +
+                    (textAnimationCounter++ % 60 === 0 ? "ㅤ" : "▮"),
```
Comment (tkjantti) [inline src/game.ts:191]: This repeating part could be in a subroutine.

**Review by tkjantti — CHANGES_REQUESTED** [review]

**Review by tkjantti — APPROVED** [review]

File: src/game.ts [hunk 4]
```diff
@@ -478,16 +490,15 @@ export const startRace = async (): Promise<void> => {
...
-    );
-    cx.restore();
-    await waitForAnyKey();
-    playTune(SFX_START);
     window.requestAnimationFrame(gameLoop);
 
-    setState(GameState.Start);
-    startRace();
+    await initialize().then(async () => {
+        setState(GameState.Init);
```
[the remark this answers is not in the document]
Comment (sz1521) [reply src/game.ts:496]: Yes, I think it was not working back for some reaso without that time but might been just hurry to get it done. I will try again witth this.

**Pull request merged**